src/main/java/com/compstore/validator/ProcessorBrandValidator.java

```diff
@@ -0,0 +1,15 @@
+package com.compstore.validator;
+
+import com.compstore.entity.ProcessorBrandEntity;
+import com.compstore.exception.EntityUsedException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProcessorBrandValidator {
+
+    public void validateDelete(ProcessorBrandEntity processorBrandEntity) {
+        if (processorBrandEntity.isUsed()) {
```

Comment:
To wstępna implementacja. Ten warunek w przyszłości przeniesiemy do innej metody (lub klasy), by móc reużyć kod